tomcat-bin/setenv.sh

```diff
@@ -42,7 +42,7 @@ else
 fi
 
 if [ "${HIGHLY_SECURE_CRYPTO_MODE_ENABLED}" == "true" ]; then
-  JAVA_OPTS="${JAVA_OPTS} -DHighSecureCryptoModeEnabled=true "
+  JAVA_OPTS="${JAVA_OPTS} -DHighSecureCryptoModeEnabled=true -Dorg.bouncycastle.fips.approved_only=true"
```

Comment:
Have we tested Cloudk deployment with this change? Can you please attach the results?